### ISSUE TITLE
Param Converter issues (lazy object extending an entity)

### ISSUE THREAD

**Issue by trsteel88** [symfony/symfony#48527]
### Symfony version(s) affected

6.2.1

### Description

I just upgraded to 6.2.1 and I'm having an issue with the param converters for some reason.

I currently have a listener firing on onKernelController which looks for a reference class within the attributes. That class is reference to a Page entity (it provides the ID for this value within a route created by the Symfony CMF Routing Bundle)

When it finds this, it creates a Proxy (using FOS LTS Proxy Manager) to that entity so it's lazy.

In the controller action if I reference the Page entity in the action arguments I get the following exception:

```
Object of class ProxyManagerGeneratedProxy\__PM__\FooBundleBundle\Entity\Page\Generatedb7b36c2f00636ed953728b9dae69f0eb could not be converted to string
```

It's as if there is a Doctrine Listener which is seeing the Proxy extends an actual Entity class and trying to do something with it and failing.

If I reference "PageInterface" which my Entity extends I don't have the issue (I think because it doesn't recognise it as an entity and try to convert it).

How can I stop whatever listener that is firing to stop this function?

**Comment by stof** [comment]
without a stack trace, it is hard to help you as we don't see where the error happens

**Comment by trsteel88** [comment]
How can I disable this functionality altogether? Not sure why it only just started happening in 6.2. I never want my doctrine entities to be auto converted.

**Comment by trsteel88** [comment]
@stof here is the stacktrace.

```
Error:
Object of class ProxyManagerGeneratedProxy\__PM__\App\Entity\Page\Generatedb7b36c2f00636ed953728b9dae69f0eb could not be converted to string

at vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:3129
at implode(' ', array('id' => object(Generatedb7b36c2f00636ed953728b9dae69f0eb)))
(vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:3129)
at Doctrine\ORM\UnitOfWork->tryGetById(array('id' => object(Generatedb7b36c2f00636ed953728b9dae69f0eb)), 'App\\Entity\\Page')
(vendor/doctrine/orm/lib/Doctrine/ORM/EntityManager.php:480)
at Doctrine\ORM\EntityManager->find('ProxyManagerGeneratedProxy\\__PM__\\App\\Entity\\Page\\Generatedb7b36c2f00636ed953728b9dae69f0eb', array(), null, null)
(vendor/doctrine/orm/lib/Doctrine/ORM/EntityRepository.php:197)
at Doctrine\ORM\EntityRepository->find(object(Generatedb7b36c2f00636ed953728b9dae69f0eb))
(vendor/symfony/doctrine-bridge/ArgumentResolver/EntityValueResolver.php:110)
at Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver->find(object(EntityManager_51e8656), object(Request), object(MapEntity), 'cmsPage')
(vendor/symfony/doctrine-bridge/ArgumentResolver/EntityValueResolver.php:57)
at Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver->resolve(object(Request), object(ArgumentMetadata))
(vendor/symfony/http-kernel/Controller/ArgumentResolver/TraceableValueResolver.php:60)
at Symfony\Component\HttpKernel\Controller\ArgumentResolver\TraceableValueResolver->resolve(object(Request), object(ArgumentMetadata))
(vendor/symfony/http-kernel/Controller/ArgumentResolver.php:54)
at Symfony\Component\HttpKernel\Controller\ArgumentResolver->getArguments(object(Request), array(object(RegisterInterestController), 'indexAction'), object(ReflectionMethod))
(vendor/symfony/http-kernel/Controller/TraceableArgumentResolver.php:39)
at Symfony\Component\HttpKernel\Controller\TraceableArgumentResolver->getArguments(object(Request), array(object(RegisterInterestController), 'indexAction'), object(ReflectionMethod))
(vendor/symfony/http-kernel/HttpKernel.php:155)
at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
(vendor/symfony/http-kernel/HttpKernel.php:74)
at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
(vendor/symfony/http-kernel/Kernel.php:184)
at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
(vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php:35)
at Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
(vendor/autoload_runtime.php:29)
at require_once('/usr/local/var/www/website/vendor/autoload_runtime.php')
(public/index.php:11)

```

**Comment by xabbuh** [comment]
Does https://github.com/symfony/symfony/pull/48489 solve your issue?

**Comment by trsteel88** [comment]
Yes, this appears to resolve the issue @xabbuh. Will I need to wait til the end of the month for this to be merged into 6.2 or would this happen sooner?

Thanks.

**Comment by nicolas-grekas** [comment]
Might happen soon :)
